app/models/user.js

```diff
@@ -33,12 +37,27 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.BOOLEAN,
         field: 'is_admin',
         defaultValue: false
+      },
+      points: {
+        type: DataTypes.INTEGER,
+        field: 'points',
+        defaultValue: 0
+      },
+      position: {
+        type: DataTypes.STRING,
+        field: 'position',
+        defaultValue: 'Developer'
       }
     },
     {
       timestamps: true,
       underscored: true
     }
   );
+  User.beforeUpdate(instance => {
+    const pos = position(instance.dataValues.points);
+    logger.info(pos);
```

Comment:
Add more information at logging. This will only print the position. There are several approaches for logging, but a descriptive line like `The position is: ${pos}`would be fine for this.

Reply:
ok